src/components/ViewingsList/_viewingsList.scss

```diff
@@ -9,7 +9,8 @@
   flex-direction: column;
   gap: 2.5rem;
   padding: 1rem;
-  width: 38rem;
+  min-width: 27rem;
```

Comment:
Curious as to why we are setting a min size of 432px, which may not be small enough for mobile. Not that our interface is ready for that now, but at some point it would be nice to support a mobile design.

Reply:
The labels go awry if it is shrunk any smaller, and mobile was not a requirement at this time.

Reply:
I think it would also be a good idea to decide on a minimum screen size we support. At my last company we supported down to 320px because that was the width of the iPhone5. Maybe we could work together with the design team to look at how far back in devices we'd like to support and make it know to the business what our minimum screen size is.

Reply:
The [design system](https://www.figma.com/proto/9jVSJUHAISjJ3A3dOUuMLi/SG%3A-Grid-%26-Spacing?page-id=5793%3A989&type=design&node-id=5793-1924&viewport=851%2C1151%2C0.35&t=3nxlMYGOAYx628tx-1&scaling=contain&starting-point-node-id=5793%3A1924) supports a breakpoint of 360px - 960px.

Reply:
Shall I remove the `min-width` or proceed to merge?

Reply:
I never sent the message. Sorry, let's remove the min width for now.